examples/ul2/src/utils/hf_prefixlm_converter.py

```diff
@@ -0,0 +1,769 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Converts Huggingface Causal LM to Prefix LM.
+
+Conversion does lightweight surgery on a HuggingFace
+Causal LM to convert it to a Prefix LM.
```

Comment:
Can we move this to composer.utils or something? If it's a general-purpose utility function that we don't expect users to modify (is it?), ideally we keep it out of Examples.

Reply:
I think that should be the plan eventually, but I don't think it's mature enough for that yet.